src/pages/wallet-import-keystore/wallet-import-keystore.ts

```diff
@@ -73,16 +120,25 @@ export class WalletImportKeystorePage {
   /**
    * Prepares the wallet based on the current key store, password and name entered by the user.
    */
-  prepareWallet(): ILocalWallet {
+  prepareWallet(type: ImportType): ILocalWallet {
     // Get the decrypted private key. We need this to retrieve the public key for the wallet.
-    let privateKey = this.keyStoreService.decryptKeyStore(this.keyStore, this.password);
+    let password = "";
+    let name = "";
+    if (type === "clipboard") {
```

Comment:
Why use separate variables for password+name when importing from file or string? I can imagine it could be possible to use the same variable for both cases. This would in turn simplify this code (and maybe other code paths in this class).

Reply:
Discussed with Mathyn, two seperate "File" and "Clipboard" inputs defined and connected to code by two different password and names. Will keep it like this for now. Future refactoring